### ISSUE TITLE
alpha matting implemented?

### ISSUE THREAD

**Issue by Tord-Zhang** [julienr/libseg#2]
Have you implement the alpha matting part?

**Comment by julienr** [comment]
No, as stated in the readme this is unfinished and only implement geodesic matting. You could fork and extend it though
